scripts/travis/integration_test.sh

```diff
@@ -9,7 +9,16 @@
 # Examples: scripts/travis/integration_test.sh
 set -e
 
+
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+ARCH=$("${SCRIPTPATH}/../archtype.sh")
+
+if [[ "${ARCH}" != "arm" ]]; then
+    curl -sL -o ~/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
+    chmod +x ~/gimme
+    eval $(~/gimme $("${SCRIPTPATH}/../get_golang_version.sh"))
+fi
+
```

Comment:
The `travis/integration_test.sh` is called only after `travis/build.sh` which already have this installed.

Reply:
> The `travis/integration_test.sh` is called only after `travis/build.sh` which already have this installed.


@tsachiherman The `travis/integration_test.sh` script is invoked directly from the .travis.yaml file.

Reply:
sorry, my bad. But given that this script is already calling `travis/build.sh` few lines below, why do we need it ?

Reply:
@tsachiherman 
Without, results in an error on the external ARM64 integration test.  
`++ go env GOPATH
./scripts/travis/integration_test.sh: line 32: go: command not found`

Reply:
@egieseke , you replaced the install that we previously had with your custom installer, and then after your custom installer doesn't install it correctly, you figure out that you need to install it over and over..
The reason I created the symbolic link when installing go was to work around setting the PATH, which cannot be done from a subshell. That's also the case here where you've seen the error coming from
```bash
GOPATHBIN=$(go env GOPATH)/bin
```
because it couldn't find `go` in any of the `PATH` locations.

Reply:
@tsachiherman Because we have multiple entry points for building and testing with Travis, it must be invoked once at the top level to set up the go environment. Note that the gimme script is also used by Travis to configure the go environment.

Reply:
@tsachiherman Removed call to gimme from integration_test.sh.